ntp-proto/src/clock.rs

```diff
@@ -240,6 +262,13 @@ impl<C: NtpClock> ClockController<C> {
     }
 
     fn set_freq(&mut self, offset: NtpDuration, last_peer_update: NtpInstant) {
+        info!(
+            freq = display(
```

Comment:
Not sure how it interacts with grouping but you can use a `%` prefix instead of `display()`.